txpool/mock.go

```diff
@@ -26,7 +26,10 @@ func (m defaultMockStore) GetBalance(types.Hash, types.Address) (*big.Int, error
 }
 
 func (m defaultMockStore) Header() *types.Header {
-	return &types.Header{}
+	//Gas limit is default gas limit for genesis block
+	return &types.Header{
+		GasLimit: 4712388,
```

Comment:
Is it possible to add a method to the `defaultMockStore` for setting the gas limit of the latest header?

Right now only 1 test uses this value I think, but in the future more unit tests may rely on this GasLimit value to be configurable